src/plugins/intel_gpu/tests/test_cases/activation_simple_gpu_test.cpp

```diff
@@ -1630,14 +1637,86 @@ TEST_P(activation_random_test, random) {
+//                                                          activation_func::sqrt,
+                                                          activation_func::elu,
+                                                          activation_func::sin,
+//                                                          activation_func::asin,
+                                                          activation_func::sinh,
+//                                                          activation_func::asinh,
+                                                          activation_func::cos,
+//                                                          activation_func::acos,
+                                                          activation_func::cosh,
+//                                                          activation_func::acosh,
+//                                                          activation_func::log,
+//                                                          activation_func::log2,
+                                                          activation_func::exp,
+                                                          activation_func::tan,
+                                                          activation_func::atan,
+//                                                          activation_func::atanh,
+                                                          activation_func::floor,
+                                                          activation_func::ceil,
+                                                          activation_func::negative,
+                                                          activation_func::negation,
+                                                          activation_func::pow,
+                                                          activation_func::reciprocal,
+                                                          activation_func::erf,
+                                                          activation_func::hard_sigmoid,
+                                                          activation_func::hsigmoid,
+                                                          activation_func::selu,
+                                                          activation_func::sign,
+                                                          activation_func::softplus,
+                                                          activation_func::swish,
+                                                          activation_func::hswish,
+                                                          activation_func::mish,
+                                                          activation_func::round_half_to_even,
+                                                          activation_func::round_half_away_from_zero,
+                                                          activation_func::gelu_tanh,
+                                                          activation_func::softsign};
```

Comment:
@wilson-seok @yeonbok as we discussed we added all the activation functions here. Commented functions are failed now so as we agreed I propose to create separate tasks for them.